build.ps1

```diff
@@ -705,6 +738,25 @@ function Build-WiXProject() {
   Invoke-Program $msbuild @MSBuildArgs
 }
 
+function Build-CompilerDependencies($Arch) {
+  Isolate-EnvVars {
+    $env:Path = "$BinaryCache\toolchains\$PinnedToolchain\PFiles64\Swift\Runtimes\0.0.0\usr\bin;${env:Path}"
+    $env:SDKROOT = "$BinaryCache\toolchains\$PinnedToolchain\PFiles64\Swift\Platforms\Windows.platform\Developer\SDKs\Windows.sdk"
+
+    Build-CMakeProject `
+      -Src $SourceCache\swift-syntax `
+      -Bin $BinaryCache\99 `
+      -InstallTo "$($Arch.ToolchainInstallRoot)\usr" `
+      -Arch $Arch `
+      -BuildTargets default `
+      -Defines @{
+        BUILD_SHARED_LIBS = "YES";
+        CMAKE_Swift_COMPILER = "$BinaryCache\toolchains\$PinnedToolchain\PFiles64\Swift\Toolchains\0.0.0+Asserts\usr\bin\swiftc.exe";
+        CMAKE_Swift_FLAGS = "-sdk ${env:SDKROOT}";
```

Comment:
Oops, another instance of needing `@("-sdk", $env:SDKROOT)`

Reply:
I uh, yes, it must've been a clone that did this!